books/RayTracingInOneWeekend.html

```diff
@@ -1229,10 +1229,16 @@
 <div class='together'>
 The rejection method presented here produces random points in the unit ball offset along the surface
 normal. This corresponds to picking directions on the hemisphere with high probability close to the
-normal, and a lower probability of scattering rays at grazing angles. The distribution present scales by the $cos^2 (\phi)$ where $\phi$ is the angle from the normal. This is useful since light arriving at shallow angles spreads over a larger area, and thus has a lower contribution to the final color.
+normal, and a lower probability of scattering rays at grazing angles. The distribution present
+scales by the $cos^2 (\phi)$ where $\phi$ is the angle from the normal. This is useful since light
```

Comment:
`$\cos^2(\phi)$` — trig functions are special terms in LaTeX, so get a backslash prefix.